# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../slmodem/2.6.16.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- slmodem-2.9.11-20051101/drivers/st7554.c.vanilla	2006-05-22 20:57:27.543494500 +0200
+++ slmodem-2.9.11-20051101/drivers/st7554.c	2006-05-22 20:57:44.516555250 +0200
@@ -1150,8 +1150,10 @@
 MODULE_DEVICE_TABLE (usb, st7554_ids);
 
 static struct usb_driver st7554_usb_driver = {
+	.driver = {
 	.owner =       THIS_MODULE,
-	.name =	       "ST7554 USB Modem",
+	.name =	       "ST7554 USB Modem"
+	},
 	.probe =       st7554_probe,
 	.disconnect =  st7554_disconnect,
 	.id_table =    st7554_ids,
